flink-runtime/src/test/java/org/apache/flink/runtime/taskexecutor/TaskExecutorSubmissionTest.java

```diff
@@ -105,30 +105,35 @@
 
 	private JobID jobId = new JobID();
 
-	private String metricQueryServiceAddress;
```

Comment:
The same as comment in `TaskExecutorTest`

Reply:
At that time, I found many duplicate code like `final ExecutionAttemptID eid1 = new ExecutionAttemptID();` in this file, so I think in each test, there should also be a unique "metricQueryServiceAddress". But actually, "metricQueryServiceAddress" is same because of "metricRegistry" is same, so it can be retained?